docs/SETUP-BUILD.md

```diff
@@ -0,0 +1,235 @@
+	ubuntu:trusty: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.
+	Digest: sha256:6cd0b1a4370ce7fbd27fddb70a5673a50c4719bd66a110b6cc8dd22a30ccb374
+	Status: Downloaded newer image for ubuntu:trusty
+	root@9164788dda01:/# 
+
+So, we are now in a container, it did work! Ctrl-P, Ctrl-Q to bail out back to the host machine. Then clean up our stuff again (change the container id so that it matches your system):
+
+	$ docker ps
+	CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
+	9164788dda01        ubuntu:trusty       "bash"              2 minutes ago       Up 2 minutes                            cranky_leakey       
+	$ docker stop 9164788dda01
+	9164788dda01
+	$ docker rm 9164788dda01
+	9164788dda01
+	$ 
+
+### Go
+Go can be downloaded [here](https://golang.org/dl/). Choose `go1.4.2.linux-amd64.tar.gz` for Ubuntu. Then go to [this page](https://golang.org/doc/install) and follow the instructions there. I've installed in `/usr/local`, as recommended. Create a local Go workspace in the home directory: 
+
+	$ mkdir -p ~/go
+
+Add the following lines to `~/.bashrc`:
+
+	export GOPATH=$HOME/go
+	export PATH=$PATH:/usr/local/go/bin:$GOPATH/bin
+
+Need to logout / login to make the changes effective (or `source .bashrc`)
+
+# Project Setup
+Created the Netplugin source tree using:
+
+	$ mkdir -p ~/go/src/github.com
+	
+Lot of good info regarding Go directory layout [here](http://golang.org/doc/code.html). Then installed the Netplugin source tree. Note that I forked the project into my own account (rschmied) on Github.
+
+	$ go get github.com/rschmied/netplugin
+	package github.com/rschmied/netplugin
+        imports github.com/rschmied/netplugin
+        imports github.com/rschmied/netplugin: no buildable Go source files in /home/netplugin/go/src/github.com/rschmied/netplugin
+	$ 
+
+### Syncing the Netplugin Fork
+In my case, my fork of Netplugin was out of date... Looking [here](https://help.github.com/articles/syncing-a-fork/) and [here](http://www.jonathanmedd.net/2013/06/git-remote-add-upstream-fatal-remote-upstream-already-exists.html) definitely helped to get to the latest master. These were my steps (and I was several commits behind which is not showing below b/c it already fetched and merged those in the meantime):
+
+	$ cd ~/go/src/github.com/rschmied/netplugin
+	$ git remote add upstream https://github.com/contiv/netplugin
+	$ git fetch upstream
+	From https://github.com/contiv/netplugin
+	 * [new branch]      mapuri/safeStateAPIs -> upstream/mapuri/safeStateAPIs
+	 * [new branch]      mapuri/stateTranslation -> upstream/mapuri/stateTranslation
+	 * [new branch]      master     -> upstream/master
+	$ git checkout master
+	Already on 'master'
+	Your branch is up-to-date with 'origin/master'.
+	$ git merge upstream/master
+	Already up-to-date.
+	$ 
+	
+### Sym Links
```

Comment:
An alternative that doesn't require creating symlinks is to checkout your fork in `$GOPATH/src/github.com/contiv/netplugin` directory as discussed here: https://github.com/contiv/netplugin/issues/43#issuecomment-91265187